rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,174 @@
+### DCP PR:
+
+# RFC Process
+
+## Summary
+
+The Human Cell Atlas Data Coordination Platform (HCA DCP) evolved informal, consensus-driven processes to coordinate and document technical decisions.
+
+This early *lightweight* and *egalitarian* approach enabled the DCP to achieve rapid velocity and incorporate a diverse set of innovative design concepts.
+
+As DCP continues to mature and scale, our community can more easily contribute and learn when there is a formal and consistent **Request for Comments (RFC)** process to propose enhancements to software or governance which are then maintained in a central repository.
```

Comment:
I think there is a lack of clarity on what is meant by governance in this context. I think software is easier to intuit as presumably a software RFC would be about requesting significant new features or changes to our existing software stack or adding a significant piece of new software to the stack. 

Governance is more nuanced so might need more spelling out.

I am certainly uncertain what might get an RFC from a governance perspective and what of our existing processes would need RFCs written about them so there is clarity on them from a community perspective.

Reply:
Here are some examples of a [Rust Governance RFC](https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md) and [Rust Roadmap](https://github.com/rust-lang/rfcs/blob/master/text/2314-roadmap-2018.md) based on this [issue](https://github.com/rust-lang/rfcs/pull/1728).

DCP PM can decide on topics for governance RFC(s).

Reply:
@lauraclarke - I've added more detail about governance to the **When is a RFC required?** section:

A RFC is required for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).